src/kernel/isr.c

```diff
@@ -0,0 +1,163 @@
+#include "idt.h"
+#include "utils.h"
+#include "../drivers/screen.h"
+#include "../drivers/IO.h"
+
+#include "isr.h"
+
+isr_handler_ptr_t interrupt_handlers[256];
+
+
+/* Can't do this with a loop because we need the address
+ * of the function names */
+void isr_install() {
+
+    set_idt_gate(0, (uint32_t)isr0);
+    set_idt_gate(1, (uint32_t)isr1);
```

Comment:
Same thing with the macro here

Reply:
not sure how to do that here...

Reply:
Don't think there is a good way to do this, especially since C preprocessor  doesn't really support loops in this stage

Reply:
done